packages/aragon-cli/src/lib/apm/grantNewVersionsPermission.js

```diff
@@ -0,0 +1,76 @@
+const APM = require('@aragon/apm')
+const ACL = require('./util/acl')
+
+module.exports = async (
+  web3,
+  apmRepoName,
+  apmOptions,
+  grantees,
+  progressHandler,
+  { gasPrice }
+) => {
+  if (grantees.length === 0) {
+    throw new Error('No grantee addresses provided')
+  }
+
+  if (progressHandler) {
```

Comment:
Maybe we should only add `progressHandler` to where they provide valuable information. As a good rule of thumb, if they are not used by our front-end, they should not be there. And `progressHandler` before the first expensive async call or after the last do not provide much value either